README.md

```diff
@@ -4,6 +4,8 @@ We define accessibility as:
 
 > the degree to which something is usable by as many people as possible
 
+Not sure where to begin? Start by reading our guide to [Getting Started With Accessibility](getting-started.md).
```

Comment:
I wonder if we should just inline the whole "getting started" doc into this README. 

_Why?_

1. It's more likely someone will read it if they didn't have to click through
2. At the moment, given the level of accessibility knowledge in the department, the goal of this repo should be to help people get started with accessibility. I think we can assume 80% of people who visit will be interested in the getting started guide. This may change as people become more familiar.

What do you think

Reply:
That sounds like a great idea to me! I'll get it moved over.